app/lib/training_route_manager.rb

```diff
@@ -40,6 +40,7 @@ def requires_study_mode?
       TRAINING_ROUTE_ENUMS[:assessment_only],
       TRAINING_ROUTE_ENUMS[:pg_teaching_apprenticeship],
       TRAINING_ROUTE_ENUMS[:opt_in_undergrad],
+      TRAINING_ROUTE_ENUMS[:hpitt_postgrad],
```

Comment:
Note to reviewer: Assuming we don't the route to be exactly like AO only at the moment